src/i18n/locale/en.json

```diff
@@ -199,17 +199,17 @@
     },
     "date": {
       "label": {
-        "today": "TODAY",
+        "today": "Today",
```

Comment:
this and Yesterday, we want to show them capitalized, this can be alternatively handled with css? @schroda

Reply:
you're right, the capitalization should be handled in the code